packages/application-shell/package.json

```diff
@@ -54,13 +54,13 @@
     "@flopflip/launchdarkly-adapter": "2.4.4",
     "@flopflip/memory-adapter": "1.1.0",
     "@flopflip/react-broadcast": "7.2.3",
-    "apollo-cache-inmemory": "1.3.10",
-    "apollo-client": "2.4.6",
-    "apollo-link": "1.2.3",
-    "apollo-link-error": "1.1.1",
-    "apollo-link-http": "1.5.5",
+    "apollo-cache-inmemory": "1.3.11",
+    "apollo-client": "2.4.7",
```

Comment:
⚠️ just we don't forget to update it in the mc-fe too otherwise we have two of them.

Reply:
Actually this is never used in the MC directly:

<img width="1379" alt="image" src="https://user-images.githubusercontent.com/1110551/49047521-f6762280-f1d7-11e8-82a1-fa52b0992112.png">